Prime-3/LiftoffCapstone
# Favorites Stuff

-added favorites list and displays the shop cards
-added functionality to favorite button so now it adds and removes shop from favorite list
-fixed footer so it only shows link to favorites and account if user is logged in
-added styling
-created shop cards for favorites list

To Test:
-log in: click favorite button on shop and then go to favorites list. Shop should be listed there. Click shop name on favorites list and it should redirect to that shop's page. Remove favorite and check list to make sure it's removed.
-logged out: make sure footer does not have favorite link or account link if not logged in
